queries.sql

```diff
@@ -1,3 +1,94 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name from animals WHERE date_of_birth >= 'Dec 31 2016' AND date_of_birth <= 'Jan 01 2019';
+SELECT name from animals WHERE neutered = 'True' and escape_attempts < '3';
+SELECT date_of_birth from animals WHERE name = 'Agumon' or name = 'Pikachu';
+SELECT name, escape_attempts from animals WHERE weight_kg > '10.5';
+SELECT * from animals WHERE neutered = 'True';
+SELECT * from animals WHERE name NOT IN ('Gabumon');
+SELECT * from animals WHERE weight_kg >= '10.4' and weight_kg <= '17.3';
+
+-- Query and Update animals TABLE
+
+BEGIN TRANSACTION;
+
+UPDATE animals
+SET species = 'unspecified'
+WHERE species IS NULL;
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+BEGIN TRANSACTION;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+SELECT * FROM animals;
+
+COMMIT;
+
```

Comment:
- [ ] Kindly make sure to add a statement to verify that the change persists after the commit in the above transaction.